artifacts/definitions/MacOS/System/QuarantineEvents.yaml

```diff
@@ -0,0 +1,35 @@
+name: MacOS.System.QuarantineEvents
+description: "This artifact parses the QuarantineEventsV2 database, which provides information
+on when a file was downloaded from the internet."
+
+type: CLIENT
+
+author: Wes Lambert - @therealwlambert
+
+parameters:
+- name: QuarantineGlob
+  default: /Users/*/Library/Preferences/com.apple.LaunchServices.QuarantineEventsV2
+
+precondition:
+      SELECT OS From info() where OS = 'darwin'
+
+sources:
+  - queries:
+     - LET QList = SELECT FullPath FROM glob(globs=QuarantineGlob)
+     - LET QEvents = SELECT * FROM sqlite(file=FullPath, query="SELECT * from LSQuarantineEvent")
+     #  Add delta (978307200 seconds between Cocoa timestamp (2020,1,1) and epoch timestamp (1970,1,1)) to provided Cocoa timestamp
```

Comment:
Is cocoa timestamp a thing? maybe we can add a parameter to the timestamp() function to support it? We current have epoch and winfiletime

Reply:
Yeah, that would be ideal -- figured I would just add it in manually for the time being, not being sure how prevalent it would be.  https://www.thecodeship.com/general/converting-cocoa-unix-timestamp/

Reply:
Yeah exactly.